app/views/articles/_liquid.html.erb

```diff
@@ -0,0 +1,14 @@
+<div class='ltag__link'>
+  <a href='<%= article.user.path %>' class='ltag__link__link'>
+    <div class='ltag__link__pic'>
+      <img src='<%= ProfileImage.new(article.user).get(150) %>' alt='<%= article.user.username + " " + "image"%>'/>
+    </div>
+  </a>
+  <a href='<%= article.path %>' class='ltag__link__link'>
+    <div class='ltag__link__content'>
+      <h2><%= title %></h2>
+      <h3><%= article.user.name %></h3>
+      <div class='ltag__link__taglist'><%= article.tag_list.map { |t| "<span class='ltag__link__tag'>##{t}</span>" }.join.html_safe %></div>
```

Comment:
Let's not generate html element via string + html_safe here.

Reply:
generating html through `<% article.tag_list.each do |t| %> <span ... > <%= t %> </span><% end %>` in recent commit